File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
'''
마이너스도 같이 묶어야 한다.
'''

def solution(s):
    number = list(map(int, s.split(" ")))
    number.sort()
    
    return str(number[0]) + " " + str(number[-1])

# def solution(s):
#     answer = ""
#     s = list(map(int,s.split(" ")))
#     min_n = str(min(s))
#     blank = " "
#     max_n = str(max(s))
#     answer = min_n + blank + max_n
#     return answer

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.Arrays;
class Solution {
    public String solution(String s) {
         String answer = "";
        String[] s_list = s.split(" ");
        int s_len = s_list.length;
        int[] new_num = new int[s_len];
        for(int i = 0; i < s_len; i++){
            new_num[i] = Integer.parseInt(s_list[i]);
        }
        
        Arrays.sort(new_num);
        
        int min_n = new_num[0];
        int max_n = new_num[s_len-1];
        
        answer = min_n + " " + max_n;


        return answer;
    }
}